webserver/rustfmt.toml

```
@@ -0,0 +1,13 @@
+max_width = 120
+hard_tabs = false
+tab_spaces = 2
+newline_style = "Unix"
+use_small_heuristics = "Default"
+reorder_imports = true
+reorder_modules = true
+remove_nested_parens = true
+edition = "2021"
```

```suggestion
edition = "2018"
```
or was it intentional?